graph-manager-docs/source/federation.mdx

```diff
@@ -123,38 +123,34 @@ server.listen().then(({ url }) => {
-  draining in-flight requests while beginning to generate query plans for incoming requests against the new config.
+These configuration files define:
 
-#### When to call `service:push`
+* Which implementing services are part of your federated graph
+* The partial schema defined by each implementing service
+* Metadata about each implementing service, including the `serviceURL` specified in the CLI command
 
-In general, you should always roll out changes in a way that is backward compatible
-(whether by modifying unused fields or by making additive schema changes)
-with existing clients to avoid downtime. Specifically, it's highly recommended to
-**call `apollo service:push` after all replicas of a federated service have deployed**.
-You don't want to deploy a new schema before the resolvers are there to support it!
-This way, you can be certain that only valid operations will be executable against
-the graph, and that operations will not be able to access fields or types that do
-not yet exist.
+When you initialize your Apollo gateway with a Graph Manager `apiKey` instead of a `serviceList`, the gateway polls Graph Manager regularly to detect and incorporate any changes to your managed configuration. When it detects a change, it continues resolving incoming requests with its existing configuration while it begins to generate query plans for the new configuration. When this process completes, the gateway begins executing new incoming requests against the new configuration.
 
-In the rare case where a schema change is _not_ backward compatible _with your
-gateway's query planner_, you should update your service registry _before_ you
-deploy your federated services. In particular, schema changes that affect query
-planning should be done as separate changes to the registry, prior to other modifications. This helps avoid the scenario where the query planner generates
-queries that fail validation in downstream services or violate resolvers'
-expectations. Some examples of this include:
+### Pushing configuration updates safely
+
+Whenever possible, you should update your service configuration in a way that is backward compatible to avoid downtime. To help ensure this, **call `apollo service:push` for an implementing service only after all replicas of the service are deployed**. This ensures that resolvers are in place for all operations that are executable against your graph, and operations can't access fields that do not yet exist.
+
+In the rare case where a configuration change is not backward compatible with your gateway's query planner, you should update your service registry _before_ you deploy your implementing services.
+
+Additionally, you should perform configuration updates that affect query planning separately from (and prior to) other changes. This helps avoid a scenario where the query planner generates queries that fail validation in downstream services or violate your resolvers.
```

Comment:
Perhaps something like "In short, when making changes that may affect the query planner, tread carefully, and consider how changes will propagate across services"?